site/plugins.json

```diff
@@ -479,6 +479,10 @@
     "repo": "https://github.com/netlify/netlify-plugin-nextjs",
     "version": "4.2.8",
     "compatibility": [
+      {
+        "version": "4.3.1",
+        "featureFlag": "build_plugins_use_prerelease"
```

Comment:
```suggestion
        "featureFlag": "build_plugins_next_js_use_prerelease"
```

Reply:
I was thinking it could be used for any prerelease plugin

Reply:
Ah, but what if you want to enable it only for a single plugin?